Document proposed call for adoption process (#68)
w3c/webrtc-charter: call-for-adoption-process.md

> * this process is meant for proposals that are candidate for adoption by the WebRTC Working Group, developed in their own repo (which typically should NOT be a @w3c repo); the said proposals are expected to be marked as CG report or Unofficial Drafts until their formal WG adoption

Reviewer: Nit: uppercase T. This runs throughout the proposed text.

> * this process is meant for proposals that are candidate for adoption by the WebRTC Working Group, developed in their own repo (which typically should NOT be a @w3c repo); the said proposals are expected to be marked as CG report or Unofficial Drafts until their formal WG adoption

Author: ```suggestion
* This process is meant for proposals that are candidate for adoption by the WebRTC Working Group, developed in their own repo (which typically should NOT be a @w3c repo); the said proposals are expected to be marked as CG report or Unofficial Drafts until their formal WG adoption.
```

> * when the Call for Adoption (CfA) starts, and until its review period ends (typically 7 days) , the repo main branch is frozen

Author: ```suggestion
* When the Call for Adoption (CfA) starts, and until its review period ends (typically 7 days) , the repo main branch is frozen.
```

> * changes that would emerge from the CfA discussion would be filed as pull requests; these pull requests can be submitted by the reviewer asking for change with prompt reviews by the proposer, or vice versa

Reviewer: voice: leaving oult "would" makes the text easier to read: "Changes that emerge from the CfA discussion, if any, must be filed as pull requests".

> * changes that would emerge from the CfA discussion would be filed as pull requests; these pull requests can be submitted by the reviewer asking for change with prompt reviews by the proposer, or vice versa

Author: ```suggestion
* Changes that emerge from the CfA, if any, discussion must be filed as pull requests; these pull requests can be submitted by the reviewer asking for change with prompt reviews by the proposer, or vice versa.
```

> * when the CfA period expires, if the Working Group chairs determine that the proposal can be adopted as is, the proposal is adopted, initial editors are named, the main branch is unfrozen and the repo transfered to the w3c github organization, with the document marked as up an Editors Draft

Reviewer: Nit: marked as up -> marked up as

> * when the CfA period expires, if the Working Group chairs determine that the proposal can be adopted as is, the proposal is adopted, initial editors are named, the main branch is unfrozen and the repo transfered to the w3c github organization, with the document marked as up an Editors Draft

Author: ```suggestion
* When the CfA period expires, if the Working Group chairs determine that the proposal can be adopted as is, the proposal is adopted, initial editors are named, the main branch is unfrozen and the repo transfered to the w3c github organization, with the document marked up as an Editors Draft.
```

> * if instead there are changes that would be required for the adoption to proceed, and assuming the proposer agrees with the said changes, the relevant pull requests get merged, and another CfA period is started to confirm the resulting document satisfies all parties, with an expectation that the reviews made during the new period would focus on the said changes.

Reviewer: Nit: confirm the resulting -> confirm the resulting
Style: "with an expectation" -> "The reviews made during the new period should focus on the changes made subsequent to the first CfA."

> * if instead there are changes that would be required for the adoption to proceed, and assuming the proposer agrees with the said changes, the relevant pull requests get merged, and another CfA period is started to confirm the resulting document satisfies all parties, with an expectation that the reviews made during the new period would focus on the said changes.

Author: ```suggestion
* If instead there are changes that would be required for the adoption to proceed, and assuming the proposer agrees with the said changes, the relevant pull requests get merged, and another CfA period is started to confirm the resulting document satisfies all parties. The reviews made during the new period would focus on the said changes.
```

> * whenever a CfA period expires without an agreement to adopt the proposal, the proposer or the chairs may decide to stop or pause the adoption review process, at which point the repo main branch gets unfrozen

Author: ```suggestion
* Whenever a CfA period expires without an agreement to adopt the proposal, the proposer or the chairs may decide to stop or pause the adoption review process, at which point the repo main branch gets unfrozen.
```
